Subject: make sure linuxfb is build when selected as plugin
From: Michael Olbrich <m.olbrich@pengutronix.de>

Without this patch the linuxfb graphics driver is not built when
1. Qt is built statically
2. the driver is selected as a plugin.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/plugins/gfxdrivers/gfxdrivers.pro |    1 +
 1 file changed, 1 insertion(+)

Index: b/src/plugins/gfxdrivers/gfxdrivers.pro
===================================================================
--- a/src/plugins/gfxdrivers/gfxdrivers.pro
+++ b/src/plugins/gfxdrivers/gfxdrivers.pro
@@ -1,6 +1,7 @@
 TEMPLATE = subdirs
 contains(gfx-plugins, ahi)          :SUBDIRS += ahi
 contains(gfx-plugins, directfb)     :SUBDIRS += directfb
+contains(gfx-plugins, linuxfb)      :SUBDIRS += linuxfb
 contains(gfx-plugins, qvfb)	    :SUBDIRS += qvfb
 contains(gfx-plugins, vnc)	    :SUBDIRS += vnc
 contains(gfx-plugins, transformed)  :SUBDIRS += transformed
